Program.cs

```diff
@@ -1,2 +1,41 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine ("Hello, World!");
+﻿// Convert to Decimal to Hexadecimal and Binary Program //
+internal class Program {
+    /// <summary>Main method</summary>
+    static void Main () {
+        Console.Write ("Input a Number : ");
+        string answer = Console.ReadLine ();
+        int decimalNumber = Convert.ToInt32 (answer);
+        DecimalToHexadecimal (decimalNumber);
+        int binary = Convert.ToInt32 (answer);
+        //Console.WriteLine(Convert.ToString(binary, 2));
+        DecimalToBinary (binary);
+    }
+
+    /// <summary>This method converts decimal to hexadecimal</summary>
+    /// <param name="decimalNumber"></param>
+    static void DecimalToHexadecimal (int decimalNumber) {
+        string hex = "";
+        while (decimalNumber > 0) {
+            int remainder = decimalNumber % 16;
+            char hexDigit = (char)(remainder < 10 ? remainder + '0' : remainder + 'A' - 10);
+            hex = hexDigit + hex;
+            decimalNumber /= 16;
+        }
+        Console.WriteLine ($"Decimal: {decimalNumber}, hexdecimal: {hex}");

```

Comment:
Trying the print variable decimalNumber here is bad idea, because this variable value gets changed inside the while loop.
If you need to print, save the decimal value to temporary variable.